src/server/client_cache.rs

```diff
@@ -155,7 +155,7 @@ pub struct ClientState {
     /// of the client.
     ///
     /// It should be included in update messages and server events to avoid needless waiting for the next init message to arrive.
-    pub(crate) change_tick: RepliconTick,
+    pub change_tick: RepliconTick,
```

Comment:
Do you need to change this one?

Reply:
I meant no the change itself, but changing the field. Was going to suggest a getter :)
But since you switched `serialize_with`, may I ask you to keep only changes related to its usage to encourage people to use `serialize_with`? If someone will find a use case for a complete custom serializer, I will be happy to open the API more.

Reply:
Okay, then could change `change_tick` into a public getter and a crate-visible setter?
Also how `get_change_limit` is used? I thought that it's only for components.